frontend/src/entities/configuration/confirm-configuration-changes-modal.tsx

```diff
@@ -133,6 +138,11 @@ export function ConfirmConfigurationChangesModal (props: ConfirmConfigurationCha
             }
         >
             <SpaceBetween size={'s'}>
+                <Alert statusIconAriaLabel='Warning' visible={isDeactivatingService} type='warning'>
+                    You are about to deactivate a service: Deactivated services will no longer appear
+                    within the MLSpace user interface. Deactivating services will terminate all active
```

Comment:
Should say "suspend" instead of "terminate". It does terminate EMR though, so not sure if that is worth noting (not possible to suspend EMR)

Reply:
If that is the case should we update the call out in the activate services page?